script.js

```diff
@@ -1,6 +1,27 @@
 /* eslint-disable linebreak-style */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable no-loop-func */
+
 const hamburger = document.querySelector('.hamburger');
 const toolbar = document.querySelector('.mobile-menu');
+const form = document.getElementById('contact-form');
+const errorMsg = document.getElementById('text');
+const inputName = document.getElementById('name');
+const inputEmail = document.getElementById('email');
+const inputMsg = document.getElementById('msg');
+
+const projectCard = [
```

Comment:
The pop-up window for the laptop does not look the same as in the design. It does not show the `See live` and `See source` buttons. It should have the same height and width as in the design and the background page should be blurry.
![pop-desktop](https://user-images.githubusercontent.com/35504149/136610899-8737717b-b014-4708-adf8-1aff3ca9338c.png)